Realme/C2/res/values/integers.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <integer name="config_defaultPeakRefreshRate">130</integer>
```

Comment:
Sorry, what?

Reply:
i just check this line i will create new pull request after editing